orttraining/orttraining/test/gradient/gradient_ops_test.cc

```diff
@@ -46,7 +47,11 @@ void GenerateRandomDataWithOneHot(
     // TODO: Consider varying mean and variance
     float scale = 5.f;
     float mean = 0.f;
+<<<<<<< HEAD
```

Comment:
conflict

Reply:
thanks, i missed that initially. fixed it